cvat/apps/engine/backup.py

```diff
@@ -905,11 +904,11 @@ def _import(importer, request, rq_id, Serializer, file_field_name, location_conf
 def get_backup_dirname():
     return settings.TMP_FILES_ROOT
 
-def import_project(request, filename=None):
+def import_project(request, queue_name, filename=None):
     if 'rq_id' in request.data:
         rq_id = request.data['rq_id']
     else:
-        rq_id = "{}@/api/projects/{}/import".format(request.user, uuid.uuid4())
+        rq_id = f"{request.user}$-api-projects-{uuid.uuid4()}-import"
```

Comment:
Is `$` typo?

Reply:
I will recommend to think about more readable names: `import~project.{uuid.uuid4()}-by-{request.user.username}`